cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -84,80 +84,83 @@ class FastRandom {
  *
  * @tparam T
  */
-template <typename T>
-concept SimpleRng = requires(T a) {
-                        {
-                            a.next()
-                            } -> std::convertible_to<uint32_t>;
-                    };
+template <typename T> concept SimpleRng = requires(T a)
```

Comment:
The changes in this file all look like format changes. Let's make sure to rebase off master once more as I'm surprised to see file changes here